docs/source/filters.rst

```diff
@@ -3,68 +3,117 @@
 Generic Filters
 ===============
 
-These filters can be applied to all policies for all resources. See the
-:ref:`Resource-Specific Filters and Actions reference <policy>` for
-resource-specific filters.
+The following filters can be applied to all policies for all resources. See the
+:ref:`Filters and Actions reference <policy>` for
+aws resource-specific filters.
 
-Value Filters
+Value Filter
 -------------
 
 Cloud Custodian provides for a flexible query language on any resource by
 allowing for rich queries on JSON objects via JMESPath, and allows for
-mixing and combining those with a boolean conditional operators that
-are nest-able. Comparison between values is configurable supporting
-scalar operators:
+mixing and combining those with boolean conditional operators that
+are nest-able. (Tutorial here on `JMESPath <http://jmespath.org/tutorial.html>`_ syntax)
+
+
+The base value filter enables the use of jmespath with data returned from a describe call.
+
+.. code-block:: yaml
+
+    filters:
+         - type: value
+           key: "State[0]"    ─▶ The value from the describe call
+           value: "running"   ─▶ Value that is being filtered against
+
+
+There are several ways to get a list of possible keys for each resource.
+
+- Via CLI
```

Comment:
also worth talking about aws cli, ie aws ec2 describe-instances ... the later part on cloud providers sounds a bit like looking at api docs. it might be worth noting as well to strip the container/envelope key for the resource.